README.md

```diff
@@ -1,6 +1,14 @@
-# Pine
-
-Pine is Kajabi's UI toolkit for building beautiful, accessible web applications.
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./docs/assets/pine-logo-dark.png">
+    <source media="(prefers-color-scheme: light)" srcset="./docs/assets/pine-logo.png">
+    <img alt="Pine: Web Component Design System" src="./docs/assets/pine-logo.png">
+  </picture>
+  <br>
```

Comment:
Could this section be refactored to two paragraph elements so the `<br> `could be dropped?